apps/cowswap-frontend/src/modules/ordersTable/utils/groupOrdersTable.ts

```diff
@@ -17,29 +17,33 @@ export function groupOrdersTable(allOrders: Order[]): OrderTableItem[] {
   const groupsMap = new Map<string, OrderTableGroupMapItem>()
 
   const orders = allOrders.reduce<ParsedOrder[]>((acc, order) => {
-    const parsedOrder = parseOrder(order)
-    const composableOrderId = order.composableCowInfo?.id
-    const parentOrderId = order.composableCowInfo?.parentId
+    try {
```

Comment:
Hmmm, are there cases when it can throw an error?

Reply:
Can't say I have reproducible steps, but when coming back to an account that had TWAPs, the app was crashing.

Probably this and the other error handling you commented might be unnecessary after this change: https://github.com/cowprotocol/cowswap/pull/3130/files/ee91d06ba99c4bc8c6dedd7c9eae295ea5f8ef77#diff-fad2963dd4bdca922b64e451c721cc472d4a61ed2c6c6392fa00983f870111b0R25

Which also shouldn't be possible if we consider the types, but still, the crash was happening because `executionInfo` was `undefined`

Reply:
Since we have the fix you mentioned I think we don't need try/catch anymore